src/libraries/System.Text.Json/tests/Common/CollectionTests/CollectionTests.Dictionary.NonStringKey.cs

```diff
@@ -5,526 +5,296 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Collections.Specialized;
-using System.IO;
 using System.Text.Json.Serialization.Metadata;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public partial class DictionaryTests
+#if !BUILDING_SOURCE_GENERATOR_TESTS
```

Comment:
Good point, we're following a similar pattern in other unsupported tests. I suspect it's because both projects are importing the `Common` source files from a shared `targets` file.